metadata.desktop

```diff
@@ -17,3 +17,7 @@ X-KDE-PluginInfo-Version=2.2
 X-KDE-PluginInfo-Category=Application Launchers
 X-KDE-PluginInfo-License=GPL-2.0+
 X-KDE-PluginInfo-EnabledByDefault=true
+
+Name[nl]=Beter applicatiemenu
+Comment[nl]=Een verbeterde vork van het Kicker-toepassingsmenu, een opstartprogramma dat is gebaseerd op trapsgewijze pop-upmenu's
```

Comment:
* Please put all the `Name`s together below the English `Name`.
* Please put all the `Comment`s together below the English `Comment`